task/buildah/0.2/buildah.yaml

```
@@ -111,6 +111,9 @@ spec:
     name: IMAGE_URL
   - description: Image reference of the built image
     name: IMAGE_REF
+  - name: SBOM_BLOB_URL
+    description: Reference of SBOM digest to enable digest-based verification from provenance
```

```suggestion
    description: Reference of SBOM blob to enable digest-based verification from provenance
```
Maybe? "Reference of SBOM digest" doesn't fully make sense to me

---

Updated it to the blob digest